cdap-docs/reference-manual/source/http-restful-api/metrics.rst

```diff
@@ -608,11 +697,10 @@ points for a metric. By default, 1 second resolution is used. Acceptable values
 above. If ``resolution=auto``, the resolution will be determined based on a time
 difference calculated between the start and end times:
 
-- ``(endTime - startTime) >= 3610 seconds``, resolution will be 1 hour; 
-- ``(endTime - startTime) >= 610 seconds``, resolution will be 1 minute; 
+- ``(endTime - startTime) > 3600 seconds``, resolution will be 1 hour; 
```

Comment:
Fixed, and added "ten hours" and "ten minutes" so the reason for the particular limits is clearer.